modules/ROOT/pages/setup.adoc

```diff
@@ -0,0 +1,196 @@
+
+_**Note to Maria** - Please add an image of the landing page of Anypoint Platform page that shows Partner Manager_
+
+NOTE: If you are unable to access Anypoint Partner Manager, please contact your MuleSoft customer success manager for assistance.
+
+* Minimum Software Requirements
+- Mule runtime 4.1.1 or higher.
+- Anypoint Studio 7.1 or higher.
+- The latest versions of the following connectors, as applicable to your use case:
+** Transport Protocols.
+*** AS2 - minimum connector version 4.0.2.
+*** SFTP - minimum connector version 1.2.4.
+** Message formats.
+*** X12 - minimum connector version 2.0.1.
+
+* You need to designate runtimes to deploy the B2B Gateway, for example, the Mule application that processes your B2B transactions. 
+
+NOTE: Ensure you have the needed runtime capacity to handle your B2B transaction load. If you are unsure about the capacity you need to handle your B2B transactions, please contact your MuleSoft representative.
+
+* A storage to persist your B2B message payloads.
+
+=== Setup Checklist
+
+Before you set up your message flows to send and receive messages with your partners, you'll need to collect information from them. You'll also need to understand how your backend systems work. In preparation, use this checklist before you start.
+
+You need the following for each trading partner:
+
+* Name of the Partner.
+* Identifiers (such as AS2 Identifier, X12 ISA, and so on).
+* Formats for the messages to send to each other (such as X12, and so on), the associated validations and the need for acknowledgment (such as TA1, 997, 999, and so on).
+* Identify of the message types to use. For example, a purchase order in X12 is an 850 mesage. Following the example, you need to know which X12 version you'll be using in your transactions (such as 004010, 005010, and so forth).
+* X.509 Certificate (For AS2 protocol for transport only).
+* The Endpoints, depending on the direction of interactions with your partner.
+
+As the host, you need the following information about your organization:
+
+* Your organization's endpoints to receive messages from partners. The endpoints for routing those messages to your backend systems. Similarly, you need the endpoints to pick up messages from your backend systems to send these to your partners.
+* Your organization's EDI message type preferences.
+* Any identifiers and certificates you intend to use.
+* Your storage information to persist the B2B payloads.
+
+== Getting Started with Anypoint Partner Manager
+
+Anypoint Partner Manager makes it easy to send and receive B2B messages with your trading partners. There are three main steps required to get started:
+
+- Associate Runtimes for Deployment
+- Allocate Partner Manager Roles
+- Configure Your Profile
+- Add and Configure Your Partners
+
+_**Maria** - Please tag these to show up on the right hand side of the doc page_
```

Comment:
Convert this to a comment adding //, or remove this line

Reply:
Thanks, this is super-helpful.